app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,81 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.model.TaskRequest;
+import com.codesoom.assignment.model.TaskResponse;
+import com.codesoom.assignment.service.TaskService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+    private final TaskService taskService;
+
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    /**
+     * 할 일 목록을 가져온다.
+     * @return ResponseEntity<List<TaskResponse>>
+     */
+    @GetMapping
+    public ResponseEntity<List<TaskResponse>> getTaskList() {
+        return new ResponseEntity<>(taskService.taskList(), HttpStatus.OK);
+    }
+
+    /**
+     * 할 일 상세 정보를 가져온다.
+     * @param id
+     * @return ResponseEntity<TaskResponse>
+     */
```

Comment:
이 메소드가 하는 일은 "할 일 상세 정보를 가져오는" 것이 아닙니다. 그건 이 메소드의 관점이에요. 메소드를 사용하는 사람의 입장에서 이 메소드의 하는 일은 "가져와서 리턴"하는 것이죠.

```suggestion
    /**
     * 주어진 식별자에 해당하는 할 일을 찾아 리턴합니다.
     * @param id 할 일 식별자
     * @return 조회된 할 일
     */
```

Reply:
Controller class javadoc 수정 및 Service class  javadoc 추가 하였습니다.